### PR TITLE
Dev refactor

### PR DESCRIPTION
**Pull request by brrt2**

### CODE REVIEW

File: client/src/main/java/building/FleetDropController.java [hunk 1]
```diff
@@ -16,29 +12,26 @@
...
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 
 public class FleetDropController implements Initializable {
 
+  private static final Logger LOGGER = Logger.getLogger(FleetDropController.class.getName());
   private static final int FIELD_SIZE = 30;
   private static final int GRID_SIZE = 10;
   private final Sea sea = new Sea();
```
Comment (rivalez) [inline client/src/main/java/building/FleetDropController.java]: better approach pass sea object in constructor (for testing)

File: client/src/main/java/building/FleetDropController.java [hunk 2]
```diff
@@ -16,29 +12,26 @@
...
+import java.util.Arrays;
+import java.util.List;
+import java.util.ResourceBundle;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 
 public class FleetDropController implements Initializable {
 
+  private static final Logger LOGGER = Logger.getLogger(FleetDropController.class.getName());
```
Comment (rivalez) [inline client/src/main/java/building/FleetDropController.java]: why LOGGER is capitalized ?

File: client/src/main/java/connection/Client.java [hunk 3]
```diff
@@ -1,23 +1,25 @@
...
 
 import java.io.IOException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 public class Client {
 
-  private static final Integer port = 5000;
+  private static final Logger LOGGER = Logger.getLogger(Client.class.getName());
+  private static final Integer PORT = 5000;
```
Comment (rivalez) [inline client/src/main/java/connection/Client.java]: why Integer? imo int is enough

File: client/src/main/java/model/Position.java [hunk 4]
```diff
@@ -28,23 +28,23 @@ static Position left(Position position) {
...
 
-  public Integer getRow() {
+  private Integer getRow() {
     return row;
   }
 
   @Override
-  public boolean equals(Object o) {
-    if (this == o) {
+  public boolean equals(Object obj) {
```
Comment (rivalez) [inline client/src/main/java/model/Position.java]: Do you hold Position in set?

File: pom.xml [hunk 5]
```diff
@@ -136,6 +139,14 @@
                     </execution>
                 </executions>
             </plugin>
+
+
```
Comment (rivalez) [inline pom.xml]: empty new lines

File: client/src/main/java/model/Position.java (hunk 4 above)
Comment (patiem) [reply client/src/main/java/model/Position.java]: Position is more abstract, every sea-field has position, every boundary, every mast etc, so sometimes we want top compare position for different types of fields.

**Pull request merged**